src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -106,26 +108,28 @@ export const AdminVenueView: React.FC = () => {
 
   return (
     <AdminRestricted>
-      <div className="AdminVenueView">
-        <div className="AdminVenueView__options">{renderAdminVenueTabs}</div>
-      </div>
-      {selectedTab === AdminVenueTab.spaces && (
-        <Spaces
-          onClickHome={navigateToHome}
-          onClickBack={navigateToHome}
-          onClickNext={navigateToTiming}
-          venue={venue}
-        />
-      )}
-      {selectedTab === AdminVenueTab.timing && (
-        <Timing
-          onClickHome={navigateToHome}
-          onClickBack={navigateToSpaces}
-          onClickNext={navigateToRun}
-          venue={venue}
-        />
-      )}
-      {selectedTab === AdminVenueTab.run && <RunTabView venue={venue} />}
+      <WithNavigationBar hasBackButton={false} withSchedule={false}>
```

Comment:
`WithNavigationBar` outside, `AdminRestricted` inside, navigation shouldn't be restricted only to admins

Reply:
Fixed in: ea4802ee1338ca4c0bf535218a1562a1d7c45bf3